packages/core/framework/src/http/express-loader.ts

```diff
@@ -72,21 +72,53 @@ export async function expressLoader({ app }: { app: Express }): Promise<{
     )
   }
 
+  let loggingMiddleware: RequestHandler
+
   /**
    * The middleware to use for logging. We write the log messages
    * using winston, but rely on morgan to hook into HTTP requests
    */
-  const loggingMiddleware = morgan(
-    IS_DEV
-      ? ":method :url ← :referrer (:status) - :response-time ms"
-      : "combined",
-    {
-      skip: shouldSkipHttpLog,
-      stream: {
-        write: (message: string) => logger.http(message.trim()),
-      },
+  if (!IS_DEV) {
+    const jsonFormat = (tokens, req, res) => {
+      const result = {
+        level: "http",
+        // Request ID can be correlated with other logs (like error reports)
+        request_id: req.requestId || "-",
+
+        // Standard HTTP request properties
+        http_verion: tokens["http-version"](req, res),
+        method: tokens.method(req, res),
+        url: tokens.url(req, res),
+
+        // Response details
+        status: Number(tokens.status(req, res)),
+        content_length: tokens.res(req, res, "content-length") || 0,
+        response_time: `${tokens["response-time"](req, res)} ms`,
+
+        // Useful headers that might help in debugging or tracing
+        referrer: tokens.referrer(req, res) || "-",
+        user_agent: tokens["user-agent"](req, res),
+
+        timestamp: new Date().toISOString(),
+      }
```

Comment:
I would like to see the `client_ip` too

Reply:
should be good @sgirones 💪 let me know what you think